examples/amp-consent/amp-consent-server.html

```diff
@@ -0,0 +1,252 @@
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+<header>
+  <div class="brand-logo">
+    PublisherLogo
+  </div>
+</header>
+<main role="main">
+  <p>This is a basic example of amp-consent where consent decision is handled by server</p>
+  <h3>Image that is blocked by '_till_responded' consent</h3>
+
+  <amp-analytics type="_ping_">
+    <script type="application/json">
+      {
+        "requests": {
+          "old": "https://fake.com/analytics?consent=CONSENT_STATE"
+        },
+        "vars": {
+          "abc": "${timestamp}.${requestCount}"
+        },
+        "triggers": {
+          "onetime": {
+            "on": "visible",
+            "request": "old"
+          }
+        },
+        "transport": {
+          "beacon": "false",
+          "xhrpost": "false",
+          "image": "true"
+        }
+      }
+    </script>
+  </amp-analytics>
+  <amp-img
+    data-block-on-consent='_till_responded'
+    src="/examples/img/bigbuckbunny.jpg" width=300 height=200></amp-img>
+
+  <h3>Image that is blocked by '_till_accepted' consent</h3>
+  <amp-img
+    data-block-on-consent='_till_accepted'
+    src="/examples/img/bigbuckbunny.jpg" width=300 height=200></amp-img>
+
+  <h3>Image that is blocked by '_auto_reject' consent</h3>
+  <amp-img
+    data-block-on-consent='_auto_reject'
+    src="/examples/img/bigbuckbunny.jpg" width=300 height=200></amp-img>
+
+  <h3>Image that is blocked by 'default' consent</h3>
+  <amp-img
+    data-block-on-consent='default'
+    src="/examples/img/bigbuckbunny.jpg" width=300 height=200></amp-img>
+
+  <h3>Image that is blocked by default (not specified) consent</h3>
+  <amp-img
+    data-block-on-consent
+    src="/examples/img/bigbuckbunny.jpg" width=300 height=200></amp-img>
+
+  <h3>Image that is NOT blocked by consent</h3>
+  <amp-img
+    src="/examples/img/bigbuckbunny.jpg" width=300 height=200></amp-img>
+
+  <amp-consent id='ABC' layout='nodisplay'>
+    <script type="application/json">{
+      "consentInstanceId": "consent-server",
+      "consentRequired": "remote",
+      "checkConsentHref": "/check-consent?consentRequired=true&consentStateValue=accepted&expireCache=true"
```

Comment:
might be worth to add a `onUpdateHref` here just for completeness sake

Reply:
Good point, I'll add one

Reply:
Done. FYI: I added `onUpdateHref` to the `amp-consent-client.html`. Because in the server example case consent is already handled by server. 

Instead `onUpdateHref` is more useful to sync client side consent change to server.